packages/hoppscotch-app/src/components/environments/teams/index.vue

```diff
@@ -143,6 +145,7 @@ const showModalImportExport = ref(false)
 const showModalDetails = ref(false)
 const action = ref<"new" | "edit">("edit")
 const editingEnvironment = ref<TeamEnvironment | null>(null)
+const editingVariableName = ref<string>("")
```

Comment:
You can drop implicit generic declaration if the type is known

```suggestion
const editingVariableName = ref("")
```